tutorials/exporting-stackdriver-elasticcloud/index.md

```diff
@@ -0,0 +1,337 @@
+
+1.  Obtain the URI of the Elasticsearch endpoint that has been provisioned. A link to this endpoint can be copied from the 
+    **Deployments** page. This value will be needed to configure Logstash output plugin configuration.
+
+    ![Copy the Elasticsearch URI](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/copy_es_uri.png)
+
+    The next section provides steps to complete the setup to send events to the new Elasticsearch deployment.
+
+## Configure the Logstash VM
+
+1.  Navigate in the GCP console to the **Compute Engine > VM Instances** page. The GCP Console provides convenient access to
+    the VM with an embedded SSH console. The example shell commands in this tutorial are concatenated, but if issues are 
+    encountered when pasting multiple commands into this console, consider switching to
+    [OS login](https://cloud.google.com/compute/docs/instances/managing-instance-access#enable_oslogin).
+
+    ![SSH to VM](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/ssh_button.png)
+
+1.  Perform typical system updates and install OpenJDK:
+
+        sudo apt-get update && \
+        sudo apt-get -y upgrade && \
+        sudo apt -y install openjdk-8-jre-headless && \
+        echo "export JAVA_HOME=\"/usr/lib/jvm/java-8-openjdk-amd64\"" >> ~/.profile && \
+        sudo reboot
+
+    After a few moments, the VM will complete its reboot and can be accessed again via SSH.
+
+## Install Logstash
+
+1.  Install logstash from Elastic.
+
+        wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add - && \
+        echo "deb https://artifacts.elastic.co/packages/6.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-6.x.list && \
+        sudo apt-get update && sudo apt-get install logstash
+
+1.  Install the Logstash Plugin for Cloud Pub/Sub. Version 1.2.0 of the Logstash Pub/Sub plugin is
+    [currently broken](https://github.com/logstash-plugins/logstash-input-google_pubsub/pull/36). Below are the steps to
+    use the previous version until version 1.2.1 is available.
+
+        cd && \
+        git clone https://github.com/logstash-plugins/logstash-input-google_pubsub.git && \
+        cd logstash-input-google_pubsub && \
+        git checkout v1.1.0 && \
+        sudo apt-get install -y rubygems && \
+        sudo gem build logstash-input-google_pubsub.gemspec && \
+        cd /usr/share/logstash && \
+        sudo bin/logstash-plugin install  ~/logstash-input-google_pubsub/logstash-input-google_pubsub-1.1.0.gem
+
+    **Expected response:**  
+
+    (after many lines...)
+
+        Validating /home/aaron/logstash-input-google_pubsub/logstash-input-google_pubsub-1.1.0.gem
+        Installing logstash-input-google_pubsub
+        Installation successful
+
+    When this issue is resolved, it should be possible to replace the git clone/checkout process above with
```

Comment:
This seems to be the last mention of specific versions of this plugin that may need to be updated.